src/App.jsx

```diff
@@ -14,33 +15,88 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+const [goods,setGoods] = useState(goodsFromServer);
+const [isAlphabetical, setIsAlphabetical] = useState(false);
+const [isByLength,setIsByLength] = useState(false);
```

Comment:
you can combine these states into one `sortType` that will receive the 'alphabetical' or `byLength` string